src/heretic/p_saveg.c

```diff
@@ -47,7 +48,8 @@ char *SV_Filename(int slot)
     filename_len = strlen(savegamedir) + strlen(SAVEGAMENAME) + 8;
     filename = malloc(filename_len);
     M_snprintf(filename, filename_len,
-               "%s" SAVEGAMENAME "%d.hsg", savegamedir, slot);
+               "%s" SAVEGAMENAME "%d.hsg", savegamedir,
```

Comment:
So, the current 7th and 8th savegame will appear on page 2?

Reply:
Correct! This way no one has to rename any existing saves to get them to show up in the list.